frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -115,6 +117,7 @@ export const AccountsSummary = ({
       ...prevBalances,
       [code]: balance.balance
     }));
+    setBalancesTrigger(Date.now());
```

Comment:
This is the part I don't like... I need the banner to be able to trigger again when there is a call to "update" (otherwise it would not show up when opening the app, since the accounts are not loaded yet, or when a tx is received that bumps us over the thresold), so I came up with this solution with some AI help. I don't love it and I am open to more suggestions. CC @thisconnect :)

Reply:
Done. I don't actually need the fiat total as discussed in another comment, but looks definitely much better